gitea/repo_commit.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 The Gogs Authors. All rights reserved.
```

Comment:
No need for gogs copyright for new code

Reply:
This code copied from gogs https://github.com/gogs/gogs/commit/ee82d35ed807b32aaf04a505e5b5260f20aaf641